src/Tizen.NUI/src/public/BaseComponents/ImageViewBindableProperty.cs

```diff
@@ -110,10 +110,31 @@ public partial class ImageView
             var imageView = (ImageView)bindable;
             if (imageView._border == null)
             {
-                // Get current properties force.
+                // Sync as current properties
                 // TODO: Need to make some flag that we only need cached property map.
+                imageView.UpdateImage();
+
+                // Get current properties force.
                 PropertyMap temp = new PropertyMap();
                 Tizen.NUI.Object.GetProperty((HandleRef)imageView.SwigCPtr, ImageView.Property.IMAGE).Get(temp);
+
+                // Update cached property map
+                if(temp != null)
+                {
+                    if(imageView._imagePropertyMap != null)
+                    {
+                        imageView._imagePropertyMap.Merge(temp);
```

Comment:
Same as here. (Actually it is deprecated API but anyway) Merge is not good here.